src/server/appserver.cpp

```diff
@@ -251,7 +256,20 @@ AppServer::AppServer(int &argc, char **argv) :
 #endif
 
     // Setup proxy
-    setupProxy();
+    if (ParametersCache::instance()->parameters().proxyConfig().type() == ProxyType::Undefined) {
+        // Migration issue?
+        LOG_WARN(_logger, "Proxy type is undefined, fix it");
+        ExitCode exitCode = ServerRequests::fixProxyConfig();
```

Comment:
Is it really appropriate to define `fixProxyConfig()` in `ServerRequests` since it does not use client/server request??

Reply:
No. It's not the only function in this case.
To do in a new PR:
https://infomaniak.atlassian.net/browse/KDESKTOP-1319?atlOrigin=eyJpIjoiZDBiM2E4ZmMxYWU0NDhmNWJlYmFiZDI0NzRiMWY2Y2QiLCJwIjoiaiJ9